contrib/sip_proxy/filters/network/source/decoder.cc

```diff
@@ -126,11 +131,13 @@ int Decoder::reassemble(Buffer::Instance& data) {
 
       ssize_t content_length_end = remaining_data.search(
           "\r\n", strlen("\r\n"), content_length_start + strlen("Content-Length:"), content_pos);
-      /* The "\n\r\n" is always included in remaining_data, so could not return -1
-      if (content_length_end == -1) {
-        break;
-      }
-      */
+
+      /**
+       * The "\n\r\n" is always included in remaining_data, so could not return -1
+       * if (content_length_end == -1) {
+       *   break;
+       * }
+       */
```

Comment:
leftover?

Reply:
keep it here on purpose to remind developers `content_length_end` should be checked. but because it has been checked before, so the code can't been invoked.